Release/include/cpprest/ws_msg.h

```diff
@@ -64,6 +64,12 @@ class websocket_outgoing_message
     /// This is useful when the client side wants to check whether the server is alive.
```

Comment:
Nice to fix old typo, change 'a the' to 'the'.
Also, I would delete the sentence 'This is useful...' because I don't think it's true. Unsolicited pongs get no response. It *is* true for pings, so makes sense where you have copied it to in the new `set_ping_message`!